source/includes/stackpath/_predefined_edge_rules.md

```diff
@@ -0,0 +1,101 @@
+### Predefined Edge Rules
+
+The predefined edge rules let you configure how StackPath responds to requests to your website. These predefined rules only work with domains that resolve to StackPath.
+
+<!-------------------- LIST PREDEFINED EDGE RULES -------------------->
+
+#### List predefined edge rules
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/predefinededgerules/dcc2771d-a524-4f8c-a666-f699985d6961"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "allowEmptyReferrer": false,
+    "forceWwwEnabled": true,
+    "id": "c988cc62-24e7-4850-9a81-95f51af0a68e",
+    "pseudoStreamingEnabled": true,
+    "referrerList": [
+      "cloudmc.io",
+      "saas.cloudmc.io"
+    ],
+    "referrerProtectionEnabled": true,
+    "robotsTxtEnabled": true,
+    "robotsTxtFile": "User-agent: *\nDisallow:",
+    "scopeId": "e9e48610-3ba6-471a-96f4-7cf18cb73455",
+    "stackId": "1f259d02-db66-4a93-8402-a0725a00a02a",
+    
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/predefinededgerules/<a href="#stackpath-site">:siteId</a></code>
+
+Retrieve the configuration of all predefined edge rules in a given [environment](#administration-environments) within a site.
+
+Attributes | &nbsp;
+------- | -----------
+`allowEmptyReferrer`<br/>*boolean* | Whether or not empty referrer is allowed.
+`forceWwwEnabled`<br/>*boolean* | Whether or not redirecting every request to a www subdomain is enabled.
+`id`<br/>*UUID* | A predefined egde rules identifier which is same as siteId.
```

Comment:
I fins this sentence ambiguous, is it the same as the SiteId or the same type (UUID).

```suggestion
`id`<br/>*UUID* | A predefined egde rules identifier which is the same as a siteId.
```
if the latter then the suggestion above is what I would have used

Reply:
It's former. The predefinedEdgeRuleId is actually the siteId as there is no entity in SP// which maps to predefined edge rule, rather it's a combination of some settings.
How about - 
`This ID is same as the siteId to which the predefined rules belong.`

Reply:
852e6a9